### PR TITLE
Host Regulations and Documents Files on S3

### PR DESCRIPTION
**Pull request by FinnIckler**
This PR hosts all wca-document files and the egulations files on S3.

### CODE REVIEW

**Comment by FinnIckler** [comment]
Changed the scope of this PR to just include the old regulations file, as the new ones are .erb files and can't be served from S3

**Comment by FinnIckler** [comment]
I think this is done pending the tnoodle releases and deploy PRs in wca-regulations and wca-regulations-translations

**Comment by FinnIckler** [comment]
I've now also done the tnoodle versions and made the other PRs. So this is ready for reviews

File: WcaOnRails/app/views/regulations/history.html.erb
Comment (gregorbg) [inline WcaOnRails/app/views/regulations/history.html.erb:1]: This file has a lot of inconsistencies:
- There is `/history/files`
- there also is `/files/history`
- there are obvious typos like `/historyfiles/2013-05-10`

File: WcaOnRails/env_config.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@
     optional :SIDEKIQ_REDIS_URL, :string, ''
     optional :DISCOURSE_URL, :string, ''
     optional :STORAGE_AWS_BUCKET, :string, ''
-    optional :STORAGE_AWS_REGION, :string, ''
+    optional :STORAGE_AWS_REGION, :string, 'us-west-2'
```
Comment (gregorbg) [inline WcaOnRails/env_config.rb:33]: Not a big fan of hardcoding this here.
Might cause a lot of scratching heads "Why am I automatically being thrown to `us-west-2` without even specifying anything?" down the line.
If you really need this default value, change it in the (checked-in!) `.env.development` file.

**Review by gregorbg — CHANGES_REQUESTED** [review]

File: WcaOnRails/env_config.rb (hunk 1 above)
Comment (FinnIckler) [reply WcaOnRails/env_config.rb:33]: makes sense! Moved it

File: WcaOnRails/app/views/regulations/history.html.erb
Comment (FinnIckler) [reply WcaOnRails/app/views/regulations/history.html.erb:1]: history/files is correct. fixed!

**Review by gregorbg — APPROVED** [review]

**Pull request merged**